administration/src/cards/CSVCard.ts

```diff
@@ -0,0 +1,83 @@
+import { format, isValid as isDateValid, parse } from 'date-fns'
+
+import { Region } from '../generated/graphql'
+import { ProjectConfig } from '../project-configs/getProjectConfig'
+import { CardBlueprint } from './CardBlueprint'
+import { ExtensionHolder } from './extensions'
+
+type HeaderMap = {
+  [header: string]: {
+    getValue: () => string | null
+    setValue: (value: string) => void
+    isValid: () => boolean
+  }
+}
+
+const IgnoreHeaders = ['Region']
```

Comment:
A user should not be able to set the region by hand. Instead only the current region should be considered. This is probably also not compatible with freinet.